data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,834 @@
+			`	A few hours later, an Avgi cruiser, a Refraktos-class Laserstar, settles down on the pad next to yours, and an Avgi officer disembarks.`
+			`	"Salutations. Hello, Captain <first> <last>. My name is Kokkino, Captain of the Distant Collision. Tell me, is the <ship> a warship, question?"`
+			choice
+				`	"Yes, the <ship> is as powerful as they come."`
+					goto warship
+				`	"No, the <ship> isn't made for combat."`
+			action
+				set "avgi: rescue intro"
+			`	"Consideration. But, are you still willing to help us in a way that takes advantage of your ship's ability to bypass the starways, question?"`
+			choice
+				`	"Yes, of course."`
+				`	"I'll hear you out."`
+				`	"I'd rather not."`
+					decline
+			`	"Approval. That is very good, for your unique capabilities may help us save the lives of some very brave people. Come, meet me in the spaceport in an hour." He gives you a set of directions to a small building in Ensemble's main spaceport before leaving. "We will have much to discuss."`
+				goto end
+
+			label warship
+			action
+				set "avgi: warship intro"
+			`	"Excellent. I am unsurprised, given its performance in the battle, but for all we know your alien vessel could be a simple freighter and yet still be formidable in our eyes. Are you willing to assist us with your warship's strength, question?"`
+			choice
+				`	"Yes, of course."`
+				`	"I'll hear you out."`
+				`	"I'd rather not."`
+					decline
+			`	"Enthusiasm. Your help will save many lives. Then, if you would, meet me in the spaceport in an hour." He gives you a set of directions to a small building in Ensemble's main spaceport before leaving. "We will have much to discuss."`
+
+			label end
+			`	Darius exits the <ship> behind you. "What was that about?" he asks.`
+			choice
+				`	"It sounds like they'd like my help with something."`
+				`	"I'm going to help them with the war effort."`
+			`	He frowns. "I see. When do you think we'll be able to head up north to look for a way out?"`
+			choice
+				`	"We'll do it soon, I promise."`
+				`	"I'm not sure."`
+			`	He looks a little disappointed, then collects himself. "Well, I suppose there's no real urgency to it. Though, I've been thinking. If your jump drive wasn't able to let you escape from the north, there might not be a point in looking in that direction."`
+			`	"What do you suggest?" you ask.`
+			`	"I have a friend among the Avgi, one who doesn't frequent the official circles in the Consonance as much. There's an entire second branch of the Tangled Shroud, mostly ran by miners and small-time shipbuilders, the kind of people who talk a lot among themselves about rumors and tall tales. The official story behind the Incursion, the invasion of the Aberrant, is that they came from a chain of systems in the East, but no one ever talks about where exactly that chain leads. I think we should pay my friend a visit after you wrap up this next mission."`
```

Comment:
At the moment, the Shroud doesn't seem to branch quite as strongly as this implies? The most direct path through it seems to go through where I'd expect this second branch to be.